### DISCUSSION TITLE
db pull command is looking for elevated permissions to read system tables

### DISCUSSION THREAD

**Discussion by jagadeeshmusali** [prisma/prisma#17408] [Q&A]
Hi,

I am using Prisma with MongoDB. When I try to db pull, 
_Error in connector: Error querying the database: Command failed (Unauthorized): not authorized on DTFPLMOS to execute command { listIndexes: "system.views"}_

The thing is we don't have permission to read these system tables. So is there a way to ignore this collection and introspect the rest of the collections?

**Comment by nurul3101** [comment]
Hello @jagadeeshmusali 👋 

At the moment, it is not possible to introspect partial collections. The only workaround I can think of is to create a clone of the database and keep the collections you have access to. You also might not need to copy over all the data for each collection, only some documents might be good enough if these documents have generally the same field structure.

Let me know if you run into any issues in trying this out.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
